### PR TITLE
Add knative-kafka release-v0.22.0

### PR DESCRIPTION
**Pull request by lberk**

### CODE REVIEW

**Comment by lberk** [comment]
cc @matzew

File: ci-operator/jobs/openshift-knative/eventing-kafka/openshift-knative-eventing-kafka-release-v0.22.0-periodics.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    path_alias: knative.dev/eventing-kafka
+    repo: eventing-kafka
+  labels:
+    ci-operator.openshift.io/prowgen-controlled: "true"
+    ci-operator.openshift.io/variant: "47"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-knative-eventing-kafka-release-v0.22.0-47-e2e-aws-ocp-47-continuous
+  spec:
+    containers:
+    - args:
```
Comment (matzew) [inline ci-operator/jobs/openshift-knative/eventing-kafka/openshift-knative-eventing-kafka-release-v0.22.0-periodics.yaml:20]: let's add a slack reporter

**Comment by matzew** [comment]
some infra / AWS issue... :pray:

**Comment by matzew** [comment]
```

INFO[2021-04-09T16:57:19Z] kafka-ch-controller-65485cb77d-2p52n    0/1   ImagePullBackOff   0     5m23s 
INFO[2021-04-09T16:57:19Z] kafka-webhook-6d55bfb495-h99vc          0/1   ImagePullBackOff   0     5m23s 
```

**Comment by matzew** [comment]
@lberk 
there is/was an issue w/ the old registry....

See here:
* https://github.com/openshift-knative/eventing-kafka/pull/145
* https://github.com/openshift-knative/eventing-kafka/pull/146

for fixes ...

**Pull request merged**